SR2020 kick-off email (#15)
srobo/volunteer-emails: SR2020/2019-08-11-sr2020-kickoff.md

Introducing the SR2020 Competition Team:

- Andy Barrett-Sprot
- Andy Busse
- Antoine Petty
- Hope Shaw
- Jake Howard
- James Seden Smith
- Jenny Fletcher
- Peter Law

There are a few different ways for you to get involved:

## The Game

We are currently looking for game design ideas for this year's competition. To submit a game proposal, please email competition-team@studentrobotics.org with a description up to about a page in length (feel free to add in diagrams if you'd like).

#### What we are looking for in a game

    - Played by 4 fully autonomous robots, but still playable by as few as 2
    - Be playable in an arena up to about 8m squared
    - Last for between 2-4 minutes
    - Have point scoring that is accessible for teams of a wide range of abilities 
    - Have an arena layout that is easily and quickly set up at the beginning of each round

The capabilities of the robots are expected to be very similar to previous years. If you are unfamiliar with previous years' competitions, rulebooks can be found [here](https://github.com/srobo/srweb/tree/master/resources) and videos of the robots competing [here](https://www.youtube.com/user/studentrobotics)

> The submission deadline is the end of Sunday 8th September. After this date all submissions will be reviewed and discussed during a meeting open to all volunteers before the game design in finalised.

Reviewer: ```suggestion
The submission deadline is noon on Saturday 7th September. On that afternoon we will be an open hosting discussion meeting on afternoon to review the submissions.
```

---
to: All Volunteers
subject: SR2020 Kick-off
---

Hello,

Hope you have all been enjoying the summer and are now ready to start helping work towards Student Robotics 2020!

> This year, we have decoupled the effort to organise the competition (Competition Team), and the effort to develop the next generation kit (Kit Team). As such, we have a new committee and are really looking forward to getting started.

Reviewer: Maybe let's update this in light of the trustees' newsletter?

Reviewer: ```suggestion
As you'll have seen from the trustees newsletter, the Competition Committee formed about a month ago. Since then we've formed an initial team and started planning for SR2020.

We'd like to introduce you to the team and outline how you can get involved.
```

We'd like to introduce you to the team and outline how you can get involved.

Introducing the SR2020 Competition Team:

- Andy Barrett-Sprot
- Andy Busse
- Antoine Petty
- Hope Shaw
- Jake Howard
- James Seden Smith
- Jenny Fletcher
- Peter Law

There are a few different ways for you to get involved:

## The Game

> We are currently looking for game design ideas for this year's competition. To submit a game proposal, please email competition-team@studentrobotics.org with a description up to about a page in length (feel free to add in diagrams if you'd like).

Reviewer: We should check that `competition-team@studentrobotics.org` isn't world readable before doing this. An alternative might be to use `game-design@studentrobotics.org` or `srobo-game@googlegroups.com` (both which have been used in the past).

Author: Would there be any disadvantage to using one of the game design specific emails? because if not it would save having to worry about the competition team email, and might speed up the email being sent?

Reviewer: Possibly. Notably I unfortunately don't know where `game-design@studentrobotics.org` actually goes (though that's likely fixable), nor whether non-member of the group can post to `srobo-game@googlegroups.com` (it's a deliberately private group for game discussions).

Reviewer 2: AFAIK `competition-team@` is an alias, so there's nothing to be seen.

`srobo-game` might be a better address, as things remain nicely in 1 place, although I don't know who as administration permissions for that

Reviewer: `competition-team@` is a group.

Reviewer: Confirmed: currently `competition-team@` is not world readable. It turns out not to archive posts, but we can live with that. Let's just stick with that.

> The submission deadline is the end of Sunday 1st September.

Reviewer: This is essentially only a week away now, do we want to allow more time? (I think we could easily push this back to the weekend of the 7th)

Reviewer 3: @JennyLynnFletcher ?
I'm happy to push it to the 7th as it's pretty late now.

Author: I think that's probably sensible

Reviewer: 6bbf10bf770daa8c4dd53248154d945cf5153f9d

#### What we are looking for in a game

    - Played by 4 fully autonomous robots, but still playable by as few as 2

>     - Be playable in an arena up to about 8m squared

Reviewer: Why "up to"?

>     - Be playable in an arena up to about 8m squared

Reviewer: Are we open to non-square arenas?

    - Last for between 2-4 minutes
    - Have point scoring that is accessible for teams of a wide range of abilities 

>     - Have an arena layout that is easily and quickly set up at the beginning of each round

Reviewer: Maybe also mention that it should ideally be something which can be reproduced in schools?

Reviewer 4: Possibly helpful: https://kb.roboticsoutreach.org/events/game-design/

Reviewer 2: Now documented in https://github.com/srobo/runbook/pull/61

> #### Capabilities of the robots

Reviewer: This section reads oddly. It feels either really specific, or rather incomplete, depending on whether you consider the reader to know what SR robots are like.

As an alternative, should we perhaps describe the rough capabilities of the robots we've seen in the past?

Reviewer: How about something like this?

Robots entered into SR are typically wheeled, about 40cm × 40cm base and able to move at about walking pace. The kit we provide enables them to add a variety of sensors including ultrasound for distance and microswitches for direct contact. These are in addition to the provided vision system which can spot 10cm × 10cm markers from a couple of meters away (or larger markers at a greater distance), which we typically put on any arena props and around the arena walls.

It might also be worth linking to some historic rulebooks for reference of what's achievable? If we do want to do this, the easiest source for them is probably the https://github.com/srobo/srweb repo (under `docs/$YEAR/rulebook.pdf`).

Reviewer: @antoinepetty @JennyLynnFletcher I think this section is important to get right. If we decide we want to assume that recipients already know about the capabilities of the robots I think that's fine, as would be changing this to remove that assumption entirely. My concern is that the current wording ends up in a middle ground which I'm concerned will end up being confusing.

Author: So putting something along the lines of: 

> The capabilities of the robots are expected to be (will be) very similar to previous years. If you are unfamiliar with previous years' competitions, rulebooks can be found here and videos of the robots competing here

Reviewer: https://github.com/srobo/volunteer-emails/pull/15/commits/f68807a67837f45930a417e65471d90282bcf10c

    - Robots are no more than 500x500x500mm at the start of the match (although may expand after the match starts)
    - Camera gives positional feedback of any fidutial markers in view

> After this date all submissions will be reviewed and discussed during a meeting open to all volunteers before the game design in finalised.

Reviewer: I found myself reading this and wondering "what date?" before realising it was above the previous sections. Maybe pull this upwards or the deadline downwards?

Reviewer: 6bbf10bf770daa8c4dd53248154d945cf5153f9d

We look forward to looking through all your game design submissions!

### Task tickets

> Soon, we will have a sizable bank of tasks which need to be completed, which you can [find on GitHub](https://github.com/srobo/tasks/issues). Each ticket there should have a small description of what needs to be done; and every task you can complete brings us one step closer to delivering another brilliant competition.

Reviewer: Can we get some milestone set up so we can link to them?

> ### Meetings

Reviewer: Maybe a more general title, since we also mention Slack here?

Perhaps this (not completely sure though)
```suggestion
### Get involved
```

If you have some questions about a task you want to work on, or would just like to get a feel for what's going on, we will be opening our competition meetings for volunteers to join. The planned times will appear on [this calendar](https://calendar.google.com/calendar/embed?src=studentrobotics.org_oqdjasvpps8smo0d5nte417rak%40group.calendar.google.com&ctz=Europe%2FLondon) and you can join in Google Meet using the link in the calendar event.

You can also [join our Slack server](https://goo.gl/forms/Maq41MHF8CYSRVn83) if you haven't already. This lets you chat with other SR volunteers and get more frequent updates on our progress.

> 

Reviewer: Are we also going to mention the srobo@ mailing list here?

Reviewer 3: I'm not sure it's worth adding it until people are able to sign up

Here's to Student Robotics 2020!

> *- The SR Team*

Reviewer: ```suggestion
*- The SR Competition Team*
```
